src/BM2/SiteBundle/Controller/ActionsController.php

```diff
@@ -943,8 +943,8 @@ public function assignedAction(Request $request) {
 				if ($soldier->getCharacter() && $soldier->getCharacter()->isInBattle()) continue;
 				if ($soldier->getCharacter() && $soldier->getCharacter()->isDoingAction('military.regroup')) continue;
 				// FIXME: can't recall from your own estates or characters, but there should also be a message:
-				if ($soldier->getCharacter() && $soldier->getCharacter()->getUser() == $character->getUser()) continue;
-				if ($soldier->getBase() && $soldier->getBase()->getOwner() && $soldier->getBase()->getOwner()->getUser() == $character->getUser()) continue;
+				// if ($soldier->getCharacter() && $soldier->getCharacter()->getUser() == $character->getUser()) continue;
+				// if ($soldier->getBase() && $soldier->getBase()->getOwner() && $soldier->getBase()->getOwner()->getUser() == $character->getUser()) continue;
```

Comment:
Because it was causing bugs- when someone spawned into a settlement, and took all the soldiers, the character who owns the settlement could not recall the soldiers, even if he should be able to. Compared to eliminating that bug, I think it's alright to comment out the anti-abuse change, since it still freezes a character for 24 hours.